### ISSUE TITLE
Parse broken when internal link and icon are on same line

### ISSUE THREAD

**Issue by meganesium2001** [progfay/scrapbox-parser#212]
**Describe the bug**

Parse broken when internal link and icon are on same line (like `Title\n[internal link][icon.icon]`.)

**To Reproduce**
Please see: https://runkit.com/meganesium2001/5f1d39cdb6e1fc001a7dca51

**Expected behavior**
```
0: Object {content: "", href: "Internal link", pathType: "relative", type: "link"}
1: Object {path: "icon", pathType: "relative", type: "icon"}
```

**Screenshots**
![image](https://user-images.githubusercontent.com/28430292/88475094-d75d0480-cf67-11ea-8316-e15bc217cfc7.png)

**Desktop (please complete the following information):**

- OS: Manjaro i3 20.0.3
- Browser Node.js
- Version 10.21.0

Thanks.

**Comment by progfay** [comment]
Thank you for your issue.

This bug is fixed on #213 , and latest package is already published!
Please update package for [v4.0.3](https://www.npmjs.com/package/@progfay/scrapbox-parser/v/4.0.3).

If you find another bug, please make an issue again!

Thanks.

**Comment by meganesium2001** [comment]
I updated the package and it works fine.

Thank you for fixing it!
